test/core/tests/resume_succeeded_step.py

```diff
@@ -0,0 +1,39 @@
+from metaflow_test import MetaflowTest, ExpectationFailed, steps
+
+
+class ResumeSucceededStepTest(MetaflowTest):
+    """
+    Resuming from the succeeded end step should work
+    """
+
+    RESUME = True
+    # resuming on a successful step.
+    RESUME_STEP = "start"
```

Comment:
it might be better to resume a "non-start" step in the test so that we can confirm we are able to skip steps successfully.

Reply:
in that scenario, we can perhaps test for the existence of an artifact generated in the start step in the resumed end step.

Reply:
done